src/recipes/mango-lassi.json

```diff
@@ -17,11 +17,6 @@
             "quantity": "",
             "measure": "",
             "ingredient": "One-Fourth cup fresh or canned mango pulp"
-        },
-        {
-            "quantity": "",
-            "measure": "",
-            "ingredient": "Three-fourth tablespoon sugar"
```

Comment:
Was this meant to be removed?

Reply:
Yes, I think was an accidental repeat of the ingredient. The author additionally indicated in the directions that you're to "sugar to taste" so the amount of sugar added before this step seems redundant anyway. I can re-commit with this added back in if necessary.

Reply:
Thanks for the clarification. I would say leave the ingredient in so we have the data as part of the ingredient list. But if you can add the "Three-fourth" to the quantity, and "tablespoon" to the measure fields.

Reply:
I'm actually working on another PR in which I go through A-P recipes and fix errors like empty measurement fields, strange white spaces, and misspellings. so for now I'll recommit with the aforementioned fixes and then submit an additional PR later on for more fixes.

Reply:
OK, sounds good.